app/views/layouts/_log_viewer.html.haml

```diff
@@ -1,4 +1,4 @@
-- legend_text ||= "Last 1000 lines from server #{@server.name} [#{@server_options[:server_id]}] in zone #{@server.my_zone}"
+- legend_text ||= _("Last 1000 lines from server %{server_name} [%{server_id}] in zone %{zone}") % {:server_name => @server.name, :server_id => @server_options[:server_id], :zone => @server.my_zone}
```

Comment:
suggest renaming `%{zone}` to `%{zone_name}` for clarity, unless this is already a pattern